examples/dem/3d-bouncing-particle/bouncing-particle-05.prm

```diff
@@ -0,0 +1,96 @@
+# Listing of Parameters
+# ---------------------
+# --------------------------------------------------
+# Simulation and IO Control
+#---------------------------------------------------
+
+subsection simulation control
```

Comment:
What is -05, -06, -07 in the filename? is it possible to have a bit more verbosity in the filenames?

Reply:
It correspond to the restitution coefficient of both the particle and the wall. --> 05 correspond to e_r = 0.5  

"bouncing-particle-restitution-coef-05.prm" would be a better file name?

Reply:
You could do a python script that can generate all the files with only one .prm

Reply:
Also, the script could launch all the simulation one after another, so no needs to launch all the simulations one by one

Reply:
@acdaigneault  is absolutely right. You can look at the mixing example which uses Jinja to do this.

Reply:
There's now a python script that generate all the .prm files.

The post processing script was modify accordingly. :)